terraform/modules/ecs/alb.tf

```diff
@@ -45,8 +45,23 @@ resource "aws_alb_listener" "http" {
   tags              = var.tags
 
 
-  default_action {
-    target_group_arn = aws_alb_target_group.polytomic.id
-    type             = "forward"
+  dynamic "default_action" {
```

Comment:
dynamic blocks are a bit limiting but this works for now

Reply:
So this just adds the redirect action if the var is `true`, right?